src/bufr/CMakeLists.txt

```diff
@@ -87,6 +87,9 @@ if ( iodaconv_bufr_ENABLED )
                        INSTALL_HEADERS LISTED
     )
 
+  set_property(TARGET atms_lib PROPERTY CXX_STANDARD 14)
```

Comment:
@PatNichols  this `14` can not be inherited from the higher level CMake?

Reply:
@BenjaminRuston No, this is the highest CMakeLists.txt in the repo. We could put these in the bundles but then it's not guaranteed it will present in general.